src/main/java/com/automationexercise/Utilities/Utility.java

```diff
@@ -199,5 +200,44 @@ public static void openWebsite(String URL) {
         getDriver().get(URL);
         implicitlyWait();
         LogUtils.info("Open website with URL: " + URL);
+    }
+        /** Get element location
+         * @param Locator the specified element
+          */
+
+    public static boolean getElementLocation(By Locator)
+    {
+        boolean isAtTop = (boolean) ((JavascriptExecutor) getDriver()).executeScript(
+                "return arguments[0].getBoundingClientRect().top <= 0", Locator);
+
+        // Output the result
+        if (isAtTop) {
+            System.out.println("Element is at the top of the page.");
+            return true;
+        } else {
+            System.out.println("Element is not at the top of the page.");
+            return false;
+        }
+    }
+
+    public static void hoverElement(WebElement Locator)
```

Comment:
You should pass the locator as **By** object instead of **Web Element** and you can find it inside the function using **findWebElement** method